addons/l10n_fr_pdp_reports/models/pdp_flow.py

```diff
@@ -0,0 +1,341 @@
+    }
+
+    name = fields.Char(
+        string="Reference",
+        required=True,
+        copy=False,
+        default=lambda self: self._get_default_name(),
+    )
+    reporting_date = fields.Date(
+        string="Reporting Date",
+        required=True,
+        help="Date associated with the aggregated reporting period.",
+    )
+    flow_type = fields.Selection(
+        selection=lambda self: self._selection_flow_type(),
+        string="Flux Type",
+        required=True,
+        default="flux10_en16931",
+        help="Identify the tax flux that generated this payload.",
+    )
+    state = fields.Selection(
+        selection=[
+            ("draft", "Draft"),
+            ("building", "Building"),
+            ("pending", "Sent"),
+            ("done", "Completed"),
+            ("error", "Error"),
+        ],
+        string="Status",
+        required=True,
+        default="draft",
+    )
+    payload = fields.Binary(
+        string="Payload",
+        attachment=True,
+        help="XML payload that is ultimately sent to the PDP API.",
+    )
+    payload_filename = fields.Char(string="Payload Filename")
+    transport_identifier = fields.Char(
+        string="Transport Identifier",
+        help="Identifier returned by the PDP transport API.",
+    )
+    transport_status = fields.Char(
+        string="Transport Status",
+        help="Raw status returned by the PDP transport API.",
+    )
+    transport_message = fields.Text(
+        string="Transport Message",
+        help="Additional message or error returned by the PDP transport API.",
+    )
+    currency_id = fields.Many2one(
+        "res.currency",
+        string="Currency",
+        required=True,
+        help="Currency of the aggregated transactions included in the payload.",
+    )
+    document_type = fields.Selection(
+        selection=[("sale", "Sale"), ("refund", "Refund")],
+        string="Document Type",
+        default="sale",
+        required=True,
+        help="Indicates whether the flow covers sales or refund transactions.",
+    )
+    transaction_type = fields.Selection(
+        selection=[
+            ("b2c", "B2C Domestic"),
+            ("international", "International B2B"),
+        ],
+        string="Transaction Type",
+        default="b2c",
+        required=True,
+        help="Identify whether the flow reports domestic B2C or international transactions.",
+    )
+    batch_index = fields.Integer(
+        string="Batch Index",
+        default=1,
+        help="Allows splitting large reporting periods into multiple payloads.",
+    )
+    last_send_datetime = fields.Datetime(
+        string="Last Send On",
+        help="Datetime of the most recent send attempt to the PDP transport.",
+    )
+    attempt_count = fields.Integer(
+        string="Send Attempts",
+        default=0,
+        help="Number of times this flow was sent to the PDP transport.",
+    )
+    company_id = fields.Many2one(
+        "res.company",
```

Comment:
I usually always add the attribute on each element, so you could add comodel_name=